test/mechanics/timeIntegration/TimeControl.cpp

```diff
@@ -38,55 +38,119 @@ BOOST_AUTO_TEST_CASE(Debug_Fuctionality)
                 };
 
     expectedMsg = "Current timestep is 0 or negative!";
-//    BOOST_CHECK_EXCEPTION(timeControl.SetTimeStepFunction([](double,double)->double{return 0.0;}),
-//                          MechanicsException,CheckException);
-
-
-
+    timeControl.SetTimeStepFunction([](TimeControl&,double,double,bool)->double{return 0.0;});
```

Comment:
Yeah you are right. Problem is, that was the old function interface and I changed it due to a hint of pHuschke. I missed that one. Strange though, that the compiler accepts it.